example.rb

```diff
@@ -4,20 +4,22 @@
 require 'posthog-ruby'
 
 posthog = PostHog::Client.new({
-   api_key: "", # You can find this key on the /setup page in PostHog
-   personal_api_key: "", # optional?
-   api_host: "https://localhost:8000", # Where you host PostHog. You can remove this line if using app.posthog.com
-   on_error: Proc.new { |status, msg| print msg }
+   api_key: "phc_EKriuIZ8en7eBMCKkkgraMERQXkVM6g2gD050z2HIqf", # You can find this key on the /setup page in PostHog
+   personal_api_key: "phx_XouNv5HTWQZkUvGkC4C8yq8cVmTI5eQ3oEkAWvniERn", # Required for local feature flag evaluation
```

Comment:
Just checking these aren't legit keys?

Reply:
Yeah, no worries there :) (they're my local dev keys, looks like they sneaked back in :P )